.travis.yml

```diff
@@ -80,6 +80,18 @@ jobs:
     - cd rasa-website
     - git commit --allow-empty -m "trigger nlu docs update"
     - git push origin master
+  - stage: Test NLU starter pack
+    python: 3.6
+    script:
+    - git clone https://github.com/RasaHQ/starter-pack-rasa-nlu.git
+    - cd starter-pack-rasa-nlu
+    - py.test test_nlu.py
+  - stage: Test stack starter pack (NLU only)
+    python: 3.6
+    script:
+    - git clone https://github.com/RasaHQ/starter-pack-rasa-stack.git
+    - cd starter-pack-rasa-stack
+    - py.test test_stack.py -k test_nlu_interpreter
```

Comment:
same here, how about putting the tests into `tests/nlu` and using that dir here?